gcn/notices/icecube/LvkNuTrackSearch.example.json

```
@@ -1,43 +1,50 @@
 {
-  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/icecube/LvkNuTrackSearch.schema.json",
+  "$schema": "https://gcn.nasa.gov/schema/stable/gcn/notices/icecube/LvkNuTrackSearch.schema.json",
```

No, actually this needs to be:

```suggestion
  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/icecube/LvkNuTrackSearch.schema.json",
```

We update the `$schema` and `$ref` fields automatically when we do a release.